setup.py

```diff
@@ -108,13 +112,15 @@ def build_extension(self, ext: CMakeExtension):
             os.makedirs(self.build_temp)
 
         subprocess.check_call(["cmake", str(ext.sourcedir)] + configure_args, cwd=self.build_temp)
-        subprocess.check_call(["cmake", "--build", ".", "--verbose"] + build_args, cwd=self.build_temp)
+        subprocess.check_call(
+            ["cmake", "--build", ".", "--verbose"] + build_args, cwd=self.build_temp
+        )
+
 
 with open(os.path.join("pennylane_lightning", "_version.py")) as f:
     version = f.readlines()[-1].split()[-1].strip("\"'")
 
 requirements = [
-    "ninja",
```

Comment:
Are all other changes besides here from new `black` version?

Reply:
Yep, everything else came with the black update.